api-client/src/maps/index.ts

```diff
@@ -84,7 +84,7 @@ export const animationDuration = 2000
 export const nameQueryParam = 'name'
 
 export const mapMarker: MarkerConfig = {
-  iconSize: 1.75,
-  symbol: 'marker_15',
-  symbolActive: 'marker_15_active'
+  iconSize: 0.75,
```

Comment:
Not sure and maybe check back with the design team but maybe the icons are a little bit small now?

Reply:
well the icons size should be increased by 10%. Are you talking about web or native?

Reply:
Ok. isn't even that smaller than before. I was just confused by `iconSize: 1.75` and `iconSize: 0.75` and thought it looks quite small now. Your choice if something has to be done here.

Reply:
no i increased it a little bit to have slightly bigger markers than the other icons on the map. We increased the svgs from 15 to 40 and these svs were converted to spriteicons. so if you compare `15*1.75 and 40*0.75` you can get the size by pixels. I think if thats not fine we gonna get the design feedback during beta